src/release/breaking-changes/index.md

```diff
@@ -18,7 +18,7 @@ release, and listed in alphabetical order:
 [Default PrimaryScrollController on Desktop]: {{site.url}}/release/breaking-changes/primary-scroll-controller-desktop
 [ThemeData's toggleableActiveColor property has been deprecated]: {{site.url}}/release/breaking-changes/toggleable-active-color
 
-### Released in Flutter 3.0
+### Released in Flutter 3
```

Comment:
Will this get confusing once we have "Released in Flutter 3.x"?

Reply:
I was following the pattern from the Flutter 2 breaking changes. Do you think 3.0 would be better?

Reply:
I suspect @sfshaza2 would prefer Flutter 3 😎

Reply:
Yes, it's "Flutter 3". This is a directive from Tim Sneath.